mne/channels/layout.py

```diff
@@ -763,7 +766,7 @@ def plot(self, scale_factor=1.5, show_names=False):
                             show_names=show_names)
 
 
-def read_montage(kind, ch_names=None, path=None, scale=True):
+def read_montage(kind, ch_names=None, path=None, transform=False):
```

Comment:
i think that it was leftover parameter based on `read_layout`. I don't think you would actually want `scale` to be applied to the actual channel positions. it makes sense when it is applied to the layout since that is for display purposes only.